spec/services/arcgis_api/geocoder_spec.rb

```diff
@@ -33,4 +33,42 @@
       )
     end
   end
+
+  describe '#find_address_candidates' do
+    it 'returns candidates from magic_key' do
+      stub_request_candidates_response
+
+      suggestions = subject.find_address_candidates('abc123')
+
+      expect(suggestions.first.to_h).to eq(
+        {
+          address: '100 Main Ave, La Grande, Oregon, 97850',
+          location: ArcgisApi::Geocoder::Location.new(longitude: -118.10754025791812, latitude: 45.328271485226445),
```

Comment:
This is probably fine. If you wanted to remove that long namespace, you could assert on `.as_json` instead of `.to_h`, and compare string keys:

```diff
diff --git a/spec/services/arcgis_api/geocoder_spec.rb b/spec/services/arcgis_api/geocoder_spec.rb
index 2763bca4f..9b9040256 100644
--- a/spec/services/arcgis_api/geocoder_spec.rb
+++ b/spec/services/arcgis_api/geocoder_spec.rb
@@ -40,14 +40,14 @@ RSpec.describe ArcgisApi::Geocoder do
 
       suggestions = subject.find_address_candidates('abc123')
 
-      expect(suggestions.first.to_h).to eq(
+      expect(suggestions.first.as_json).to eq(
         {
-          address: '100 Main Ave, La Grande, Oregon, 97850',
-          location: ArcgisApi::Geocoder::Location.new(longitude: -118.10754025791812, latitude: 45.328271485226445),
-          street_address: '100 Main Ave',
-          city: 'La Grande',
-          state: 'OR',
-          zip_code: '97850',
+          'address' => '100 Main Ave, La Grande, Oregon, 97850',
+          'location' => { 'longitude' => -118.10754025791812, 'latitude' => 45.328271485226445 },
+          'street_address' => '100 Main Ave',
+          'city' => 'La Grande',
+          'state' => 'OR',
+          'zip_code' => '97850',
         },
       )
     end
```